src/api.ts

```diff
@@ -92,23 +113,43 @@ router.post('/logout', (req, res) => {
 router.get('/allRoles', getAllRoles);
 
 // Members
-router.get('/allMembers', allMembers);
-router.post('/setMember', setMember);
-router.post('/deleteMember', deleteMember);
+router.get('/allMembers', async (req, res) => {
+  let handled = await allMembers(req, res);
+  res.status(handled.status).json(handled);
+});
+
+router.get('/getMember/:email', async (req, res) => {
+  let handled = await getMember(req, res);
+  res.status(handled.status).json(handled);
+});
+
+router.post('/setMember', async (req, res) => {
+  let handled = await setMember(req, res);
+  res.status(handled.status).json(handled);
+});
+
+router.post('/deleteMember', async (req, res) => {
```

Comment:
It might be better to have this endpoint as `router.delete` instead of `router.post` to be more consistent with REST routes. What do you think?